gameplay/src/Vector4.inl

```diff
@@ -55,6 +55,12 @@ inline const Vector4 Vector4::operator/(const float x) const
     return Vector4(this->x / x, this->y / x, this->z / x, this->w / x);
 }
 
+inline Vector4& Vector4::operator/=(float x)
+{
+    scale(1.0f / x);
```

Comment:
You're going to get slighting less accurate results this way. I suggest using `this->x /= x;`. This is what other vector math libraries do, like [glm](http://glm.g-truc.net/0.9.7/index.html).